Dockerfile

```diff
@@ -1,13 +1,21 @@
-FROM node:12-alpine
+FROM node:12-alpine as builder
```

Comment:
can you explain the "as builder" syntax?

Reply:
> @theneva Do you know how this might affect the build script provided by you?
> 
> https://github.com/Unleash/unleash-docker/blob/290d698221811e2dd63f2845d368624b067e5233/tools/docker-utils.js#L24-L29

I'll take a look tomorrow 🙂

Reply:
@ivarconr 
Basically it's used to reduce images size.

See multistage builds docs:
https://docs.docker.com/develop/develop-images/multistage-build/``